7_react/react1/week1/src/App.js

```diff
@@ -0,0 +1,53 @@
+
+import './App.css';
+
+function App() {
+  return (
+    <div className="react-app">
+  <ToDoList/> 
+    </div>
+  );
+}
+
+export default App;
+
+const todo = [
+  { activity: "Get out of bed", date: "Wed Sep 13 2017"},
+  { activity: "Brush teeth", date: "Thu Sep 14 2017"},
+  { activity: "Eat breakfast", date: "Fri Sep 15 2017"},
+  ];
+
+
+  function Todo(props) {
+    const date = props.date; 
+    const activity = props.activity;
+   return(
+    <div>
+    <span className="activity-color">{activity} </span> 
+    <span className="date-color">{date}</span>
+  </div>
+   )
+}
```

Comment:
Good job figuring out how to generate the UI dynamically based on the todo array